Casks/accordance.rb

```diff
@@ -9,8 +9,8 @@
   homepage "https://www.accordancebible.com/"
 
   livecheck do
-    url "https://accordancefiles1.com/xml/appcast_#{version.major}.xml"
-    strategy :sparkle
+    url "https://accordancebible.com/accordance-installers/"
+    regex(%r{href=.*?/Accordance_([0-9]+(\.[0-9]+)+)\.dmg}i)
```

Comment:
```suggestion
    regex(/href=.*?/Accordance[._-]v?(\d+(?:\.\d+)+)\.dmg/i)
```
Same reason as https://github.com/Homebrew/homebrew-cask/pull/125694#discussion_r897507247